source-repo-scripts/source_changelog.bash

```diff
@@ -1,23 +1,19 @@
 #!/bin/bash
 # bash source_changelog.bash <PREV_VER>
 
-PREV_TAG=$1
+PREV_VER=$1
 
 git fetch --tags
 
-REPO=$(basename `git rev-parse --show-toplevel`)
-REPO="${REPO/ign-/gz-}"
-MAJOR=${PREV_TAG%.*.*}
-BRANCH=${REPO/sdformat/sdf}${MAJOR}
-TAG="${REPO}${MAJOR}"
+REPO=$(basename $(git remote get-url origin))
```

Comment:
Not really working as we expect:
```
+++ git remote get-url origin
++ basename git@github.com:ignitionrobotics/ign-cmake.git
+ REPO=ign-cmake.git
```

producing:
```
https://github.com/gazebosim/ign-cmake.git/pull/309
```

Reply:
wow, I've been sloppy here. I think https://github.com/gazebo-tooling/release-tools/pull/826/commits/50e0efb8d57505fc351fee1522b0f1e667e679e0 will fix it?

Reply:
> wow, I've been sloppy here. I think https://github.com/gazebo-tooling/release-tools/commit/50e0efb8d57505fc351fee1522b0f1e667e679e0 will fix it?

yep thanks!